rnd/autogpt_server/autogpt_server/data/execution.py

```diff
@@ -243,20 +245,34 @@ async def upsert_execution_input(
 async def upsert_execution_output(
     node_exec_id: str,
     output_name: str,
-    output_data: Any,
+    output_data: str,  # JSON serialized data.
 ) -> None:
     """
     Insert AgentNodeExecutionInputOutput record for as one of AgentNodeExecution.Output.
     """
     await AgentNodeExecutionInputOutput.prisma().create(
         data={
             "name": output_name,
-            "data": json.dumps(output_data),
+            "data": output_data,
             "referencedByOutputExecId": node_exec_id,
         }
     )
 
 
+async def update_graph_execution_stats(graph_exec_id: str, stats: dict[str, Any]):
```

Comment:
This will be auxiliary column for execution analytics

Reply:
Is this actually str, json?

Reply:
yes

Reply:
Is it possible to type it like that?

Reply:
you mean the db column type? yes once @aarushik93 killed the sqlite. but not at this point.